Fluid/Filters/NumberFilters.cs

```diff
@@ -19,6 +22,43 @@ public static FilterCollection WithNumberFilters(this FilterCollection filters)
             return filters;
         }
 
+        public static FluidValue Abs(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            return NumberValue.Create(Math.Abs(input.ToNumberValue()), true);
```

Comment:
Why the true flag for integral? Add a unit test with a decimal value to prove it's wrong

Reply:
`Abs()` can be applied to `double` too, I just add few test cases to prove that's work

Reply:
I was talking abou the "the true flag". The fact that the unit test worked might be a bug with this flag actually.

Reply:
Why the flag should be `false`?!!

Reply:
I think the best way to "keep" the integral property consistent would be to flow its value in the call. Something like this:
```
var integral = (input is NumberValue numberValue && numberValue.IsIntegral;
return NumberValue.Create(Math.Abs(numberValue.ToNumberValue()), integral);
```